installing/installing_gcp/installing-gcp-user-infra-vpc.adoc

```diff
@@ -46,7 +46,8 @@ include::modules/installation-gcp-project.adoc[leveloffset=+2]
 include::modules/installation-gcp-enabling-api-services.adoc[leveloffset=+2]
 include::modules/installation-gcp-limits.adoc[leveloffset=+2]
 include::modules/installation-gcp-service-account.adoc[leveloffset=+2]
-include::modules/installation-gcp-permissions.adoc[leveloffset=+3]
+include::modules/installation-gcp-permissions.adoc[leveloffset=+2]
+include::modules/minimum-required-permissions-upi-gcp.adoc[leveloffset=+2]
```

Comment:
@xenolinux Testing and discussions regarding minimum permissions for shared VPC is still ongoing. Please remove this section. cc @jianli-wei

Reply:
@xenolinux @arkadeepsen Maybe we can insert the **Required GCP roles** section for the IPI XPN installation scenario by https://github.com/openshift/openshift-docs/pull/54885. Thanks!